src/Sylius/Bundle/CoreBundle/Console/Command/InstallCommand.php

```diff
@@ -26,25 +26,29 @@ final class InstallCommand extends AbstractInstallCommand
     /** @var array<int, array<string, string>> */
     private array $commands = [
         [
-            'command' => 'check-requirements',
+            'command' => 'sylius:install:check-requirements',
             'message' => 'Checking system requirements.',
         ],
         [
-            'command' => 'database',
+            'command' => 'sylius:install:database',
             'message' => 'Setting up the database.',
         ],
         [
-            'command' => 'setup',
+            'command' => 'sylius:install:setup',
             'message' => 'Shop configuration.',
         ],
         [
-            'command' => 'jwt-setup',
+            'command' => 'sylius:install:jwt-setup',
             'message' => 'Configuring JWT token.',
         ],
         [
-            'command' => 'assets',
+            'command' => 'sylius:install:assets',
             'message' => 'Installing assets.',
         ],
+        [
+            'command' => 'cache:clear',
+            'message' => 'Loading sample data.',
```

Comment:
I suppose it should be more like `Clearing cache` 😄

Reply:
Oh my bad 😅 , nice catch!